packages/e2e/test/web-extension/dapp-sdk/features/buildExpiredValidityIntervalTx.ts

```diff
@@ -0,0 +1,36 @@
+import { EMPTY, catchError, take, tap } from 'rxjs';
```

Comment:
@VanessaPC It seems these files are duplicated.
packages/e2e/test/web-extension/dapp-sdk/features/setExpiredValidityIntervalTx.ts
packages/e2e/test/web-extension/dapp-sdk/features/buildExpiredValidityIntervalTx.ts

Reply:
Thanks, they are as of a rebase gone wrong, I'll delete.